src/mono/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ project(mono)
 
 include(../../eng/native/configurepaths.cmake)
```

Comment:
while you're here:

```suggestion
include(${CLR_ENG_NATIVE_DIR}/configurepaths.cmake)
```

Reply:
`configurepaths` defines `CLR_ENG_NATIVE_DIR`